nservicebus/rabbitmq/configuration-api.md

```diff
@@ -14,197 +14,242 @@ To use RabbitMQ as the underlying transport:
 
 snippet:rabbitmq-config-disable-callback-receiver
 
-NOTE: In Versions 4 and above Callbacks are disabled by default. To enable them follow [Callbacks documentation](/nservicebus/messaging/handling-responses-on-the-client-side.md#message-routing-version-6-and-above).
-
 This means that the queue will not be created and no extra threads will be used to fetch messages from that queue.
 
-By default 1 dedicated thread is used for the callbacks. To add more threads, due to a high rate of callbacks, use the following:
+By default, 1 dedicated thread is used for the callbacks. To add more threads, due to a high rate of callbacks, use the following:
 
 snippet:rabbitmq-config-callbackreceiver-thread-count
 
 
-### Controlling the message ID strategy
+### Versions 4 and above
+
+In Versions 4 and above, callbacks no longer directly managed by the RabbitMQ transport and are not enabled by default. To enable them, follow the  [Callbacks documentation](/nservicebus/messaging/handling-responses-on-the-client-side.md#message-routing-nservicebus-callbacks-version-1-and-above).
+
+
+## Transport Layer Security support
+
+In Versions 4 and above, the RabbitMQ transport supports creating secure connections to the broker using Transport Layer Security (TLS). For information on how to configure TLS on the RabbitMQ broker, refer to the [RabbitMQ documentation](http://www.rabbitmq.com/ssl.html). To enable TLS support, set the `UseTls` setting to true in the connection string. If the RabbitMQ broker has been configured to require client authentication, a client certificate can be specified in the `CertPath` setting. If that certificate requires a password, it can be specified in the `CertPassphrase` setting.
+
+An example of these settings being set in a connection string via code:
+
+snippet:rabbitmq-connection-tls
+
+Or configuration:
 
-By default NServiceBus uses the `message-id` property of the AMQP standard to relay the message id. If this header isn't set the transport will throw an exception since NServiceBus needs a message ID in order to perform retries, de-duplication etc. in a safe way. In integration scenarios where the sender is not controlled consider using a custom scheme that extracts the message ID from e.g.a custom header or some data contained in the actual message body. In these cases in a custom strategy by calling:
+snippet:rabbitmq-connection-tls-config
+
+
+## Controlling the message ID strategy
+
+By default, NServiceBus uses the `message-id` property of the AMQP standard to relay the message ID. If this header isn't set, the transport will throw an exception, because NServiceBus needs a message ID in order to perform retries, de-duplication etc. in a safe way. For integration scenarios where the sender is not controlled, consider using a custom scheme that extracts a message ID from a custom header or some data contained in the actual message body. This custom strategy can be configured by calling:
```

Comment:
in this context, `because NServiceBus needs a message identifier` reads better than `because NServiceBus needs a message ID` because it's already specified, the actual, `message-id` property much earlier.